index.html

```diff
@@ -0,0 +1,6349 @@
+<!Doctype html>
+<html>
+<title>Canvas Palette</title>
+<meta charset='utf-8'>
+<head>
+    <title>imgExample</title>
+    <script src="data/4x4.json"></script>
+    <script src="data/32x32.json"></script>
+    <link href="style.css" rel="stylesheet"/>
+</head>
+<body>
+<header>
+    <h1>CodeJam - Pallete</h1>
+</header>
+<main>
+    <div>
+        <div class="menu">
+        <section class="section-1">
+            <ul>
+                <li>Paint bucket</li>
+                <li>Choose color</li>
+                <li>Move</li>
+                <li>Transform</li>
+            </ul>
+        </section>
+        <section class="section-2">
+            <ul>
+                <li>Current color</li>
```

Comment:
As i understand, you add color as a background, but in the future, you need to change color using js, so you need to change way of adding color